lib/mayaUsd/ufe/UsdSceneItemOps.h

```diff
@@ -48,8 +46,9 @@ class MAYAUSD_CORE_PUBLIC UsdSceneItemOps : public Ufe::SceneItemOps
 
     void             setItem(const UsdSceneItem::Ptr& item);
     const Ufe::Path& path() const;
-    inline UsdPrim   prim() const
+    inline PXR_NS::UsdPrim prim() const
     {
+        PXR_NAMESPACE_USING_DIRECTIVE
```

Comment:
Macros like TF_AXIOM have no namespace, so forced to use the using directive here.